bpf/kmesh/workload/sockops_tuple.c

```diff
@@ -49,7 +50,7 @@ static inline void build_auth_key(struct bpf_sock_ops *skops, struct bpf_sock_tu
 }
 
 // clean map_of_auth
-static inline void clean_auth_map(struct bpf_sock_ops *skops)
```

Comment:
why change it

Reply:
Sorry, clean_auth_map is modified by mistake when the function name is modified. As described below, clean_send_map= > clean_dstinfo_map should be modified.